src/App.tsx

```diff
@@ -20,12 +23,13 @@ export const App: React.FC = () => {
                 id="search-query"
                 className="input"
                 placeholder="Type search word"
+                onChange={(e) => setMovieTitle(e.target.value)}
               />
             </div>
           </div>
         </div>
 
-        <MoviesList movies={moviesFromServer} />
+        <MoviesList movies={moviesFromServer} movieTitle={movieTitle}/>
```

Comment:
Ensure that the `MoviesList` component is correctly implemented to filter the movies based on the `movieTitle` prop. If the component does not handle filtering internally, you might need to filter the `moviesFromServer` array before passing it to `MoviesList`.